File: resources/C/findPrime.c
#include <stdio.h>

int isPrime(int n);

int main()
{
    for (int i = 100; i < 501; i++)
    {
        if(isPrime(i))
        {
            printf("%i\n", i);
        }
    }
}

int isPrime(int n)
{
    for (int x = 2; x * x <= n; x++)
    {
        if (n % x == 0)
        {
            return 0;
        }
    }
    return 1;
}

File: resources/Java/FindPrime.java
import java.util.Scanner;

public class FindPrime {
    public static boolean isPrime(int x) {
        for (int i = 2; i <= Math.sqrt(x); i++) {
            if (x % i == 0) {
                return false;
            }
        }
        return true;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();
        for (int i = 2; i <= x; i++) {
            if (isPrime(i)) {
                System.out.print(i + " ");
            }
        }
    }
}

